src/ui/buttons/button.jsx

```diff
@@ -262,7 +262,9 @@ export function Button({
   const shouldShowWalletMenu =
     isWallet && instrument && showWalletMenu({ instrument, userIDToken });
   const borderRadiusClass =
-    borderRadius >= 0 ? CLASS.BORDER_RADIUS : `${CLASS.SHAPE}-${shape}`;
+    typeof borderRadius === "number"
```

Comment:
What happens here if borderRadius is a negative number?

Reply:
It gets caught and throws an error here! https://github.com/paypal/paypal-checkout-components/pull/2323/files#diff-527159ccc01dff03b42b4c3d556d5ca3a8a13963e63e0f0ffc64784316c79bdaR688-R693